### PR TITLE
fix .examples/apache/docker-compose.yml

### PR DESCRIPTION
**Pull request by the0ffh**
added missing version
applied proper indentation

### CODE REVIEW

**Comment by pierreozoux** [comment]
Thanks for your contribution!

File: .examples/apache/docker-compose.yml [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
+    volumes:
     - ./mysql/runtime2:/var/lib/mysql
-  environment:
+    environment:
     - MYSQL_DATABASE=db
     - MYSQL_ROOT_PASSWORD=CHANGE_ME
     - MYSQL_USER=app
     - MYSQL_PASSWORD=CHANGE_ME
+    ports:
+    - '3306:3306'
```
Comment (crazy-max) [inline .examples/apache/docker-compose.yml]: Why expose mysql ?

Comment (mattab) [reply .examples/apache/docker-compose.yml]: @crazy-max Maybe you can open a PR to remove again these two lines?

Comment (crazy-max) [reply .examples/apache/docker-compose.yml]: Done #119 ;)

**Pull request merged**